.github/workflows/nightly.yml

```diff
@@ -27,7 +27,9 @@ jobs:
 
       - run: cargo install --force --path risc0/cargo-risczero
```

Comment:
We should drop this

Reply:
The validator uses the installed version: https://github.com/risc0/risc0/blob/main/tools/crates-validator/src/lib.rs#L211-L212

Reply:
I do see what you mean, we could call `cargo run` inside the validator or expect the local `./target/` exists to better match up with the local vs installed. I am not sure the right path though.

Reply:
Ah, I didn't realize we used it like this. Maybe we could pass an env var from CI so that the validator could pick it up.